tests/SimplifyTest.elm

```diff
@@ -18526,7 +18526,8 @@ maybeAndThenTests =
         [ test "should not report Maybe.andThen used with okay arguments" <|
             \() ->
                 """module A exposing (..)
-a = Maybe.andThen f x
+a0 = Maybe.andThen f x
+a1 = Maybe.andThen << Just -- f arg missing
```

Comment:
I think in general I would prefer these kinds of "don't report error given specific conditions" to be in their own tests that expect no errors, I feel like that would be better. But no need to change it now